src/ape/api/projects.py

```diff
@@ -366,7 +366,7 @@ def __repr__(self):
     def __ape_extra_attributes__(self) -> Iterator[ExtraModelAttributes]:
         yield ExtraModelAttributes(
             name=self.name,
-            attributes=self.contracts,
+            attributes=lambda: self.contracts,
```

Comment:
It knows to eval a function to obtain attribute names every time? Or does it once and caches? (if the later, it should be represented in a comment)

Reply:
it does once and caches

Reply:
but ya, realizing it shouldnt do it that way since it may be cached already, so ill fix that